lib/std/child_process.zig

```diff
@@ -668,7 +668,7 @@ pub const ChildProcess = struct {
             if (any_ignore) os.close(nul_handle);
         }
         if (any_ignore) {
-            try windows.SetHandleInformation(nul_handle, windows.HANDLE_FLAG_INHERIT, 0);
+            try windows.SetHandleInformation(nul_handle, windows.HANDLE_FLAG_INHERIT, windows.HANDLE_FLAG_INHERIT);
```

Comment:
Hard to catch and not your fault, but this looks like a unnecessary race condition, because we dont create a (forcefully named on Windows) pipe.
The proper way to do it is to use the securityattribute, which the process spawn can directly use:
https://stackoverflow.com/questions/2344887/select-which-handles-are-inherited-by-child-process/63890166#63890166
Windows has another api to use arbitrary handles, which is even more complex, with the shortcomings explained in #14251.

Reply:
Thanks for pointing that out, I did not think of a case of race condition.

Reply:
You did not mention in your commit that this prevents potentially leaks to other parallel spawned processes, which would inherit the opened `/Device/Null` filehandle. Otherwise, you can mark this as resolved (if it works).